llama_stack/templates/bedrock/run.yaml

```diff
@@ -85,6 +85,8 @@ providers:
     config:
       api_key: ${env.TAVILY_SEARCH_API_KEY:=}
       max_results: 3
+      timeout: 30.0
+      connect_timeout: 10.0
```

Comment:
Looks like a lot of outdated `connect_timeout` fields in the run.yaml and docs above. This field was in a previous version of this PR but since removed, right?